ansible/psi/README.md

```diff
@@ -0,0 +1,36 @@
+Resources for setting up CI in PSI OpenStack
+============================================
+
+We have a "front-door-ci" project on the PSI OpenStack clouds rhos-d and rhos-01. The latter's nested KVM support and performance is good enough for our tasks runners.
+
+Access
+------
+Sign into the OpenShift web console on https://api.rhos-01.prod.psi.rdu2.redhat.com with group "redhat.com", your RedHat user name, and your Kerberos password. You need to be in the [front-door-ci group](https://rover.redhat.com/groups/group/front-door-ci) for this.
+
+The only change that was, and currently needs to be done manually is to add SSH and ICMP to the "default" security group. This step may be automated in the future.
+
+Automation setup
+----------------
+
+You need to install the OpenStack SDK, either with `sudo dnf install python3-openstacksdk` (on Fedora), or `pip install openstacksdk`.
+
+After that, you need to set up `~./config/openstack/clouds.yml`. If you don't already have one, copy or symlink [clouds.yml](./clouds.yml), otherwise merge it with your existing one. You need to add your credentials to `~/.config/openstack/secure.yaml`, see the comment in [clouds.yml](./clouds.yml) for how to do that.
```

Comment:
it should be `~/.config/openstack/clouds.yml` (dot is before `/`)

Reply:
While editing this, can you please rewrite `You also need to add your credentials to...` (adding `also`) as first I misunderstood that you only need to do it if you already have `clouds.yml`. Or put it on separate line?

Reply:
You mean the two "clouds.yml" links? The link targets are supposed to lead to the template in git. See https://github.com/cockpit-project/cockpituous/blob/psi-openstack/ansible/psi/README.md , they seem to be working as intended. Or is there something wrong with my `~./config/openstack/clouds.yml`? (then I don't see it, sorry)